data_curation_app.py

```diff
@@ -160,7 +157,7 @@ def hide_anchor_link():
             next_page()
 
         add_empty_space(5)
-        col5, col6 = st.columns([6, 1])
+        col5, col6 = st.columns([5, 1])
```

Comment:
For later : it would be great to have variables for those figures. And we could use the width of the invoice to determine this. It would allow us to work with "landscape" format as well.